storage-access.bs

```diff
@@ -286,11 +291,11 @@ ISSUE(privacycg/storage-access#137): Add links to current entry and session hist
 
 <h3 id="storage">Changes to various client-side storage mechanisms</h3>
 
-ISSUE(privacycg/storage-access#4): Should this API affect client-side storage other than cookies?
+<h4 id="cookies">Cookies</h4>
 
-ISSUE(privacycg/storage-access#31): Write this section. For each kind of client-side storage affected, modify them to invoke [=determine if a site has storage access=] & modify their behavior based on the result.
+This API is intended to be used with environments and user agent configurations that block access to unpartitioned cookies in a [=third party context=]. At the time of this writing, this concept has not yet been integrated into the [=HTTP-network-or-cache fetch=] and {{Document/cookie}} algorithms. To allow for such an integration, the [=cookie store=] will need to be modified to receive information about the top-level and embedded site of the request (to determine whether to attach cross-site, partitioned, or no cookies) as well as whether the request was made for a document that has storage access, through running the [=determine if a site has storage access=] algorithm that is defined in this specification.
 
-<h4 id="cookies">Cookies</h4>
+Once the cookie store allows for receiving information about storage access, we would update [=HTTP-network-or-cache fetch=] and [=document.cookie=] to run [=determine if a site has storage access=] and pass this information to the [=cookie store=] when retrieving cookies.
```

Comment:
I think this could be a bit clearer about this only exposing `SameSite=None` cookies of the site that has storage access.

(Also the cookie store would need access to the top-level regardless of this as it needs that for `SameSite` handling.)

Reply:
Does this match what Safari and Firefox are shipping today? (I don't think so, but maybe I'm wrong)

Reply:
Firefox and Safari presumably include cookies that lack a `SameSite` attribute. But I would expect `SameSite=Strict` to excluded regardless, for instance.

Reply:
@annevk 's understanding matches my own.

Reply:
Ok, this feels like:

```
When getting unpartitioned cookies from the cookie store through storage access, user agents MUST follow applicable SameSite restrictions (i.e. not attach cookies marked SameSite=Strict or SameSite=Lax in third-party contexts).
```